test/e2e_node/cpumanager_test.go

```diff
@@ -1102,12 +1098,8 @@ var _ = SIGDescribe("CPU Manager", ginkgo.Ordered, ginkgo.ContinueOnFailure, fra
 						siblingsCPUs := makeThreadSiblingCPUSet(cpus)
 						gomega.Expect(pod).To(HaveContainerCPUsEqualTo(cnt.Name, siblingsCPUs))
 
-						// expect full uncore cache worth of cpus to be assigned to uncoreCacheID equal to 1
-						// since CPU0 is part of reserved CPUset, resulting in insufficient CPUs from
-						// uncoreCacheID equal to 0
-						expUncoreCPUSet, err := uncoreCPUSetFromSysFS(1)
-						framework.ExpectNoError(err, "cannot determine shared cpus for uncore cache on node")
-						gomega.Expect(pod).To(HaveContainerCPUsEqualTo(cnt.Name, expUncoreCPUSet))
+						// expect full uncore cache worth of cpus to be assigned from the same uncore cache block
+						gomega.Expect(pod).To(HaveContainerCPUsWithSameUncoreCacheID(cnt.Name))
```

Comment:
ok, we need also to check this uncore ID is different from the uncore ID the reserved CPUs are affine to, because this was previously captured by the test.

Reply:
Or not? We totally do if we have strict-cpu-reservation set, that's fine. And this was added and it works. But in this case I don't remember if we want to enforce this.